.ci/scripts/Makefile

```
@@ -5,8 +5,10 @@
 BUILD=../../build
 PACKAGE_STORAGE=.package-storage
 PACKAGE_STORAGE_PATH=$(BUILD)/$(PACKAGE_STORAGE)
-PACKAGE_STORAGE_BRANCH=update-apm-$(shell date "+%Y%m%d%H%M%S")
+PACKAGE_STORAGE_BRANCH = $(eval PACKAGE_STORAGE_BRANCH := $$(shell date "+%Y%m%d%H%M%S"))$(PACKAGE_STORAGE_BRANCH)
```

```suggestion
PACKAGE_STORAGE_BRANCH:=update-apm-$(shell date "+%Y%m%d%H%M%S")
```

I think we'll always expand that variable? So may as well just evaluate it immediately, and keep the Makefile a bit simpler.